### PR TITLE
Refactor: HAL-02 Secure key phrases

### PR DESCRIPTION
**Pull request by hoailinh1210**
### Summary (Please recap what you changed or fixed)
- Splitting mnemonic phrases to small and don't save original recovery phrases on state/global state

### Checklist (If you won't pass this checklist please comment why)

-   [x] I removed all commented or unnecessary code.
-   [ ] Provide the screenshots due to UI changed or bug fixed
-   [ ] My code has covered these test cases (please list down your tested cases):

### CODE REVIEW

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -6,7 +6,9 @@ name: Node.js CI
 on:
     pull_request:
         branches: [develop, master]
-
```
Comment (djhahe) [inline .github/workflows/check.yml:9]: pls remove this

File: client/package.json [hunk 2]
```diff
@@ -55,6 +55,7 @@
 		"redux-thunk": "^2.3.0",
 		"reselect": "^4.0.0",
 		"sass": "^1.26.10",
+		"shareable-seed": "^2.0.4",
```
Comment (djhahe) [inline client/package.json:58]: pls use fixed version `"shareable-seed": "2.0.4"`

File: client/package.json [hunk 3]
```diff
@@ -136,5 +137,8 @@
 		"webpack-cli": "^4.9.1",
 		"webpack-dev-server": "^4.15.0",
 		"webpack-merge": "^5.8.0"
+	},
+	"resolutions": {
+		"secrets.js-grempe": "https://github.com/34r7h/secrets.js"
```
Comment (djhahe) [inline client/package.json:142]: Do we still need resolution ?

**Review by djhahe — CHANGES_REQUESTED** [review]

File: client/package.json (hunk 3 above)
Comment (hoailinh1210) [reply client/package.json]: Changed new package

**Review by djhahe — APPROVED** [review]

**Pull request merged**